### PR TITLE
Reverse intake

### LINKED ISSUE #92: Button for reversing intake
Add a button to reverse the intake in case notes get stuck

### PR DESCRIPTION
**Pull request by Jack-Haefele**
closes #92 
created method for reversing intake in intake subsystem
bound left trigger to reversing intake command

### CODE REVIEW

**Comment by willitcode** [comment]
Add a description of what you did to your PR description please.

File: src/main/java/frc/robot/commands/ReverseIntake.java
Comment (willitcode) [inline src/main/java/frc/robot/commands/ReverseIntake.java:1]: Rename to RunIntakeReverse to comply with formatting. (See: commands must always start with an action verb)

File: src/main/java/frc/robot/RobotContainer.java [hunk 1]
```diff
@@ -137,6 +138,7 @@ public class RobotContainer {
   private FireNoteRoutineNoLimitSwitch fireNoteRoutineNoLimitSwitch =
       new FireNoteRoutineNoLimitSwitch();
   private RunIntake runIntake = new RunIntake();
+  private ReverseIntake reverseIntake = new ReverseIntake();
```
Comment (willitcode) [inline src/main/java/frc/robot/RobotContainer.java:141]: Remember to rename the variable when you rename the class.

File: src/main/java/frc/robot/subsystems/Intake.java [hunk 2]
```diff
@@ -41,6 +41,11 @@ public void runIntake() {
     intakeUpper.set(Constants.Intake.INTAKE_MOTOR_SPEED);
   }
 
+  /** Runs the intake motors in reverse. */
+  public void reverseIntake() {
```
Comment (willitcode) [inline src/main/java/frc/robot/subsystems/Intake.java:45]: Also rename this to runIntakeReverse

**Review by willitcode — CHANGES_REQUESTED** [review]

**Review by Berdenson — COMMENTED** [review]
Looks good to me. Only issue I can see is possibly lessening the speed/power of the reverse intake. I don't know if we want it at full blast. (Don't apply anything I say until Quinn or Gabe say smth about it).

**Review by willitcode — APPROVED** [review]
Looks good to me. I'm not worried about making the power different unless we discover in testing that we have to.

**Comment by willitcode** [comment]
@QuackitsQuinn @Jack-Haefele Gotta run spotless, then I can push the big green button

File: src/main/java/frc/robot/ekwgfnwekjgnwek
Comment (willitcode) [inline src/main/java/frc/robot/ekwgfnwekjgnwek:1]: uhhhh maybe delet this? or at least don't include it in this PR?

**Review by willitcode — CHANGES_REQUESTED** [review]

**Review by willitcode — APPROVED** [review]

**Pull request merged**